test/MUXControlsTestApp/MSTest/MUXControlsTestApp.csproj

```diff
@@ -4,6 +4,11 @@
   <PropertyGroup>
     <UnitTestPlatformVersion Condition="'$(UnitTestPlatformVersion)' == ''">$(VisualStudioVersion)</UnitTestPlatformVersion>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
```

Comment:
Leftover from trying to create Store packages?
We could leave in. I manually deleted before my last diff.

Reply:
The AppxBundle:Never part is needed because otherwise the Debug_test configuration generates .appxbundle and then the MSTest integration in VS needs to re-build the project every time.